test/leveldb.test.js

```diff
@@ -983,7 +983,7 @@ _testAllAlgorithms(function collision(algo, t) {
         pnodes: ['a', 'a'],
         vnodes: NUMBER_OF_VNODES,
         backend: fash.BACKEND.LEVEL_DB,
-        location: '/tmp/' + uuid.v4()
+        location: '/var/tmp/' + uuid.v4()
```

Comment:
I had some issues running out of memory on my zone when running the tests, and it seems that `/var/tmp` helped.  I couldn't get all the tests to work, but this helped the tests get farther without complaining about `ENOMEM`

Reply:
Can we dig into the test failures to try to solve them? Or is that a science project?

Reply:
i'll check it out today and see how deep it goes